simulation/dynamic_model_quad_diag.hpp

```diff
@@ -253,14 +253,14 @@ static inline dynamic_model_quad_diag_conf_t dynamic_model_quad_diag_default_con
     conf.air_density            = 1.2f;                 ///< Air density
 
     //default home location (EFPL Esplanade)
-    conf.home_coordinates[X]    = 46.51852236174565f;   ///< Latitude of the simulation home waypoint
-    conf.home_coordinates[Y]    = 6.566044801857777f;   ///< Longitude of the simulation home waypoint
-    conf.home_coordinates[Z]    = 400.0f;               ///< Altitude of the simulation home waypoint
+    conf.origin.latitude    = 46.51852236174565f;   ///< Latitude of the simulation home waypoint
```

Comment:
don't you want to use the define of constant.hpp not to have to update this everywhere in the code as discussed above